variables.tf

```diff
@@ -9,6 +9,14 @@ variable "log_bucket_id" {
   type        = string
 }
 
+
+variable "logging" {
+  description = "object containing access bucket logging configuration."
```

Comment:
add here, that target_bucket and target_prefix need to be supplied.
And default, it should enable like how it was doing before.